src/fit/blueprints/wod_blueprint.py

```diff
@@ -0,0 +1,23 @@
+from flask import Blueprint, jsonify
+from ..services.rabbitmq_service import rabbitmq_service
+from ..services.user_service import get_all_users
+
+wod_bp = Blueprint('wod_bp', __name__)
+
+@wod_bp.route("/generate", methods=["POST"])
+def generate_wods_for_all_users():
+    users = get_all_users()
+    count = 0
+
+    for user in users:
+        message = {
+            "user_email": user.email,
+            "exercise_ids": []  # Empty for now
+        }
```

Comment:
You should define a message format for communication so that if the format changes your code will break before production.